training_record/serializers.py

```diff
@@ -62,14 +62,30 @@ class ReadOnlyRecordSerializer(serializers.ModelSerializer):
     role = serializers.IntegerField(
         source='event_coefficient.role',
         read_only=True)
+    allow_ordinary_user_review = serializers.BooleanField(read_only=True,
+                                                          default=False)
+    allow_department_admin_review = (
+        serializers.SerializerMethodField(read_only=True))
+    allow_school_admin_review = (
+        serializers.SerializerMethodField(read_only=True))
     off_campus_event = OffCampusEventSerializer(read_only=True)
     campus_event = CampusEventSerializer(read_only=True)
 
     class Meta:
         model = Record
         fields = ('id', 'create_time', 'update_time', 'campus_event',
                   'off_campus_event', 'user', 'status', 'contents',
-                  'attachments', 'status_str', 'feedback', 'role', 'role_str')
+                  'attachments', 'status_str', 'feedback', 'role', 'role_str',
+                  'allow_department_admin_review', 'allow_school_admin_review',
+                  'allow_ordinary_user_review')
+
+    def get_allow_department_admin_review(self, obj):
+        '''Get status of whether department admin can review or not.'''
+        return obj.status == Record.STATUS_SUBMITTED
+
+    def get_allow_school_admin_review(self, obj):
+        '''Get status of whether school admin can review or not.'''
+        return obj.status == Record.STATUS_DEPARTMENT_ADMIN_APPROVED
```

Comment:
Should be able to review as long as status IS in (DEPARTMENT APPROVED, SCHOOL REJECTED, SCHOOL APPROVED)